devel/python/python/ert/job_queue/driver.py

```diff
@@ -56,34 +80,36 @@ def submit( self, name, cmd, run_path, argList, num_cpu=1, blocking=False):
         argc = len(argList)
         argv = (ctypes.c_char_p * argc)()
         argv[:] = map(str, argList)
-        job_c_ptr = cfunc.submit(self, cmd, num_cpu, run_path, name, argc, argv)
+
+        job_c_ptr = self._submit(cmd, num_cpu, run_path, name, argc, argv)
```

Comment:
Since you have reimplemented `job` then you can have the `submit` function return it directly